cmd/libsnap-confine-private/tool.c

```diff
@@ -79,9 +82,9 @@ void sc_call_snap_update_ns(int snap_update_ns_fd, const char *snap_name, struct
 
     /* Switch the group to root so that directories, files and locks created by
      * snap-update-ns are owned by the root group. */
```

Comment:
the comment doesn't sound quite right anymore,  invoke while switching... or something like that?

Reply:
I've tweaked the wording